meta-ibm/recipes-phosphor/webui/webui-vue/changeLogo.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
Should add a "-e" so that if we do hit the error path where brand is not available, we just exit the script. Currently we just go the default path and mount the blank logo on error path.

Reply:
Done on new PR.